examples/plugins/afk.js

```diff
@@ -0,0 +1,50 @@
+// Plugins are loaded using the bot.loadPlugin() function
+// The load plugin function excepts one function as its only parameter
+// The plugin runs the function passing it two parameter
+// The former is the bot object and the latter is the bot option
+// the options contain version player info and other plugins
+// Here is an example AFK plugin
+
+// Each plugin should have a inject function
+// This is the function that will be put in bot.loadPlugin
+function inject (bot, option) {
+// It is a good idea to make a namespace for your plugin
+  bot.afk = {}
+
+  let afkInterval, rotation
+  // This namespace can be used to store some variuabels for user reffrence.
+  bot.afk.status = 'Idel'
+
+  // All your plugin functions should be in the name space other than the logic.
+  bot.afk.start = async function () {
+    afkInterval = setInterval(async () => {
+      if (rotation === 0) {
+        await bot.look(0, 0)
+        rotation = true
+      } else {
+        await bot.look(Math.PI, 0)
+        rotation = false
+      }
+    }, 3000)
+    bot.afk.status = 'afk'
```

Comment:
imo the bot.afk.status shouldn't be polluting the bot object, it should be a variable scoped within the plugin

Reply:
This can be used as a reference which is quite convenient. 
Let's say you are AFK and you don't know that. Then you try to walk with pathfider, it would cause issues if you keep looking away from the path you are trying to walk.
So I think it is a good idea to let future devs know the importance of reffrence

Reply:
wouldn't it be better to have a getter then, so they can't mutate the status and cause problems?

Reply:
this is only supposed to be an small example so I see no need for a getter.